chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -758,3 +769,77 @@ pub fn people_kusama_local_testnet_config() -> Result<Box<dyn ChainSpec>, String
 		.build(),
 	))
 }
+
+// PeoplePolkadot
+fn people_polkadot_genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> serde_json::Value {
+	serde_json::json!({
+		"balances": people_polkadot_runtime::BalancesConfig {
+			balances: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| (k, PEOPLE_POLKADOT_ED * 4096 * 4096))
+				.collect(),
+		},
+		"parachainInfo": people_polkadot_runtime::ParachainInfoConfig {
+			parachain_id: id,
+			..Default::default()
+		},
+		"collatorSelection": people_polkadot_runtime::CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: PEOPLE_POLKADOT_ED * 16,
+			..Default::default()
+		},
+		"session": people_polkadot_runtime::SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                         // account id
+						acc,                                 // validator id
+						people_polkadot_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		"polkadotXcm": {
+			"safeXcmVersion": Some(SAFE_XCM_VERSION),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this. `aura: Default::default()`
+	})
+}
+
+fn people_polkadot_local_genesis(para_id: ParaId) -> serde_json::Value {
+	crate::system_parachains_specs::people_polkadot_genesis(
+		// initial collators.
+		invulnerables(),
+		testnet_accounts(),
+		para_id,
+	)
+}
+
+pub fn people_polkadot_local_testnet_config() -> Result<Box<dyn ChainSpec>, String> {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 0.into());
+	properties.insert("tokenSymbol".into(), "DOT".into());
+	properties.insert("tokenDecimals".into(), 10.into());
+
+	Ok(Box::new(
+		PeoplePolkadotChainSpec::builder(
+			people_polkadot_runtime::WASM_BINARY.expect("PeoplePolkadot wasm not available!"),
+			Extensions { relay_chain: "polkadot-local".into(), para_id: 1004 },
```

Comment:
Can i somehow already double-check this number?

Reply:
Not yet, chicken and egg :). It will be part of the call to `force_register` the para, which takes both the Wasm blob and the `ParaId` to which to register it.